src/MOI/callbacks/MOI_heuristic.jl

```diff
@@ -0,0 +1,103 @@
+function moi_heuristic_xprs_optnode_wrapper(func::Function, model::Optimizer, callback_data::CD) where {CD<:CallbackData}
```

Comment:
`::Function` there blocks specialization, see: https://docs.julialang.org/en/v1/manual/performance-tips/#Be-aware-of-when-Julia-avoids-specializing

Reply:
Let me see if I get it: Since we will be calling the exact same function repeatedly, it might be better to specialize the wrapper code to it, right?

Reply:
Just use `::Function`. You shouldn't need to worry about this level of performance tuning yet.

Reply:
I am sorry but I will have to disagree with @odow and require the proper implementation

Reply:
I consider `::Function` a bad practice when it is not strictly necessary

Reply:
Ok, removed